backend/src/databasefunctions.ts

```diff
@@ -0,0 +1,80 @@
+import { PoolConfig, ClientConfig } from 'pg';
+
+const { Pool } = require('pg');
+require('dotenv').config();
+
+const pgconfig: PoolConfig = {
+  user: process.env.DBUSER,
+  database: process.env.DATABASENAME,
+  password: process.env.DBPASSWORD,
+  host: process.env.DBHOST,
+  port: parseInt(process.env.DBPORT),
+};
+
+const pool = new Pool(pgconfig);
+
+pool.on('error', (err: Error, client: ClientConfig) => {
+  console.error(err, client);
+});
+export const createTable = async (sqlStatement: string) => {
+  if (
+    sqlStatement &&
+    !sqlStatement.toLowerCase().includes('create table')
+  ) {
+    throw new Error('wrong type of SQL statement');
+  }
+  let client;
+  try {
+    client = await pool.connect();
+    const res = await client.query(sqlStatement);
+    return res;
+  } finally {
+    try {
+      client.release();
+    } catch (err) {
+      console.error(err);
+    }
+  }
+};
```

Comment:
Same as the comment above - there is a lot of similar code in the two other functions. Is it possible to pull it out to a single function taking in arguments?

Reply:
That's a good point, i can probably rewrite the code today.

I stole the basis for this code, so i haven't thought very much about it but i agree.